Content.Server/Atmos/Piping/Binary/EntitySystems/GasVolumePumpSystem.cs

```diff
@@ -219,5 +220,18 @@ private void OnPacketRecv(EntityUid uid, GasVolumePumpComponent component, Devic
                     return;
             }
         }
+
+        private void OnMapInit(EntityUid uid, GasVolumePumpComponent pump, MapInitEvent args) // Frontier - Enable on map init
```

Comment:
can you put the comment above the function rather than on the side of it for better readability? same for the other systems